src_c/pgcompat.h

```diff
@@ -195,6 +195,11 @@
 
 
 #if defined(SDL_VERSION_ATLEAST)
+
+#if !(SDL_VERSION_ATLEAST(2, 0, 0))
+/* SDL 1.x does not have SDL_Window */
+typedef void SDL_Window;
```

Comment:
If this solves a compile error, I suspect it replaces it with a lot of "unused variables" warnings (as I presume we cannot call anything that sets that variable nor pass it own to any function to consume that value).  If SDL 1 support is a long term goal for us, then I think we ought to keep that part of the code base for warning free as well and as such I would "vote" for a "proper warningfree fix" here.

Reply:
Good point. Thanks.